system/service.py

```diff
@@ -403,10 +403,31 @@ def get_service_tools(self):
                 self.svc_initscript = initscript
 
         def check_systemd():
-            return os.path.exists("/run/systemd/system/") or os.path.exists("/dev/.run/systemd/") or os.path.exists("/dev/.systemd/")
+
+            # tools must be installed
+            if location.get('systemctl',False):
+
+                # this should show if systemd is the boot init system
+                # these mirror systemd's own sd_boot test http://www.freedesktop.org/software/systemd/man/sd_booted.html
+                for canary in ["/run/systemd/system/", "/dev/.run/systemd/", "/dev/.systemd/"]:
```

Comment:
I think the absolute correct way would be to check for `systemd/system`, because systemd might be running as user, so `systemd/user` could exist, but `systemd/system` doesn't.